src/replica/replica_disk_migrator.cpp

```diff
@@ -148,12 +166,153 @@ bool replica_disk_migrator::check_migration_args(const replica_disk_migrate_requ
     return true;
 }
 
-// TODO(jiashuo1)
 // THREAD_POOL_REPLICATION_LONG
-void replica_disk_migrator::migrate_replica(const replica_disk_migrate_request &req) {}
+void replica_disk_migrator::migrate_replica(const replica_disk_migrate_request &req)
+{
+    if (status() != disk_migration_status::MOVING) {
```

Comment:
When will this condition happen? could you please take an example for me?

Reply:
Now hasn't  this condition, add it only make sure the process must under `MOVING`.

Reply:
OK, I suggest that you can consider this condition in your further pull request.